diff --git a/node_modules/recyclerlistview/src/platform/reactnative/scrollcomponent/ScrollComponent.tsx b/node_modules/recyclerlistview/src/platform/reactnative/scrollcomponent/ScrollComponent.tsx
index 0d0dedc..676af89 100644
--- a/node_modules/recyclerlistview/src/platform/reactnative/scrollcomponent/ScrollComponent.tsx
+++ b/node_modules/recyclerlistview/src/platform/reactnative/scrollcomponent/ScrollComponent.tsx
@@ -83,7 +83,7 @@ export default class ScrollComponent extends BaseScrollComponent {
                 horizontal={this.props.isHorizontal}
                 onScroll={this._onScroll}
                 onLayout={(!this._isSizeChangedCalledOnce || this.props.canChangeSize) ? this._onLayout : this.props.onLayout}>
-                <View style={{ flexDirection: this.props.isHorizontal ? "row" : "column" }}>
+                <View style={{ flexDirection: this.props.isHorizontal ? "row" : "column", flexGrow: 1 }}>
                     {renderContentContainer(contentContainerProps, this.props.children)}
                     {this.props.renderFooter ? this.props.renderFooter() : null}
                 </View>
