diff --git a/node_modules/recyclerlistview/dist/reactnative/platform/reactnative/scrollcomponent/ScrollComponent.js b/node_modules/recyclerlistview/dist/reactnative/platform/reactnative/scrollcomponent/ScrollComponent.js
index 2f7033c..4a8a731 100644
--- a/node_modules/recyclerlistview/dist/reactnative/platform/reactnative/scrollcomponent/ScrollComponent.js
+++ b/node_modules/recyclerlistview/dist/reactnative/platform/reactnative/scrollcomponent/ScrollComponent.js
@@ -102,7 +102,7 @@ var ScrollComponent = /** @class */ (function (_super) {
         //     ...props,
         // } = this.props;
         return (React.createElement(Scroller, __assign({ ref: this._getScrollViewRef, removeClippedSubviews: false, scrollEventThrottle: this.props.scrollThrottle }, this.props, { horizontal: this.props.isHorizontal, onScroll: this._onScroll, onLayout: (!this._isSizeChangedCalledOnce || this.props.canChangeSize) ? this._onLayout : this.props.onLayout }),
-            React.createElement(react_native_1.View, { style: { flexDirection: this.props.isHorizontal ? "row" : "column" } },
+            React.createElement(react_native_1.View, { style: __assign({ flexDirection: this.props.isHorizontal ? "row" : "column" }, this.props.scrollViewProps.contentContainerStyle) },
                 renderContentContainer(contentContainerProps, this.props.children),
                 this.props.renderFooter ? this.props.renderFooter() : null)));
     };
